src/verifiers/documentStatus/didSigned/didSignedDocumentStatus.test.ts

```diff
@@ -985,5 +881,106 @@ describe("verify", () => {
         }
       `);
     });
+    it("should pass when DID document is signed and is not revoked by an OCSP", async () => {
+      whenPublicKeyResolvesSuccessfully("0x1245e5B64D785b25057f7438F715f4aA5D965733");
+
+      const handlers = [
+        rest.get(
+          "https://ocsp.example.com/0x69e1a174ea67e1c3119639f713f8a7348bbda54fdce60903621398cc2fea4d40",
+          (_, res, ctx) => {
+            return res(
+              ctx.json({
+                revoked: false,
+                documentHash: "0x69e1a174ea67e1c3119639f713f8a7348bbda54fdce60903621398cc2fea4d40",
+              })
+            );
+          }
+        ),
+      ];
+
+      const server: SetupServerApi = setupServer(...handlers);
+      server.listen();
+
+      const res = await openAttestationDidSignedDocumentStatus.verify(didSignedOcspResponderV3, options);
+      expect(res).toMatchInlineSnapshot(`
+        Object {
+          "data": Object {
+            "details": Object {
+              "issuance": Object {
+                "did": "did:ethr:0x1245e5B64D785b25057f7438F715f4aA5D965733",
+                "issued": true,
+              },
+              "revocation": Object {
+                "address": "https://ocsp.example.com",
+                "revoked": false,
+              },
```

Comment:
i was trying around v2 and v3 i noticed that v3 returns `issuance` and `revocation` as object, compared to v2's as array

Reply:
Yes I noticed that too. It is most likely because OA v3 doesn't allow multiple issuers

https://www.openattestation.com/docs/docs-section/roadmap/v3/major-changes#issuer